src/admin-toolkit.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, IEngine, PointerEventsSystem } from '@dcl/ecs'
+import { ReactBasedUiSystem } from '@dcl/react-ecs'
+import { getComponents, IPlayersHelper, ISDKHelpers } from './definitions'
+import { createAdminToolkitUI } from './admin-toolkit-ui'
+
+// Create a system to manage the AdminToolkit
+export function createAdminToolkitSystem(
+  engine: IEngine,
+  pointerEventsSystem: PointerEventsSystem,
+  reactBasedUiSystem: ReactBasedUiSystem,
+  sdkHelpers?: ISDKHelpers,
+  playersHelper?: IPlayersHelper,
+) {
+  let adminToolkitEntity: Entity | null = null
```

Comment:
since this var is local to this fn, it is possible for someone to create multiple admin toolkit systems. idk if that's a problem we should care or not but just pointing it out